CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 ### 1.0.3 (Next)
 
-#### Features
+#### Fixes
```

Comment:
Uhm what happened here? Should be features. Generally we don't need a CHANGELOG for updating documentation so I think you should just revert this.